### ISSUE TITLE
Live usb dosn't see my ssd

### ISSUE THREAD

**Issue by maloczek** [imperador/chromefy#56]
HI i am trying to install chromefy on toshiba chromebook, i generated the image by the method 1 and made the live usb but when i try to install the chrome os from the live usb it doesn't see my ssd when i run fdisk -l i only get my usb. Maybe for chromebook there is a better way to install chromefy

**Comment by denzilferreira** [comment]
It may be that your SSD is seen as nvme0 ?

**Comment by jlukanc1** [comment]
I have this same issue on my Asus T100. I can't find anything that will list the internal SSD drive name.

**Comment by skandavc18** [comment]
Even I have the same issue. my Nvme SSD is not being recognized. It is not even listed in lsblk

**Comment by jlukanc1** [comment]
Turns out, the SSD died :/
And it was soldered into the tablet...

**Comment by skandavc18** [comment]
No. But Windows 10 is installed on it and it is running perfectly fine.

**Comment by denzilferreira** [comment]
Remember that you need to use the SSD in non-raid. You can change the sata
mode in the bios of the laptop.

D

**Comment by skandavc18** [comment]
Thanks a Lot. Now it is recognized.
